common-lib/src/cmr/common/util.clj

```diff
@@ -947,3 +947,13 @@
   (let [index (.indexOf s substring)]
    (when (>= index 0)
     index))))
+
+(defn scrub-token
+  "Cut out the middle of the token, leaving 10 characters on each end.
+  For the token that contains less than 10 characters(like tokens in the test),
+  remove the last character."
+  [token]
+  (let [token-length (.length token)]
+    (if (<= 10 token-length)
+      (str (subs token 0 10) (subs token (- token-length 10) token-length))
```

Comment:
For something with less than 20 characters - this would contain all the characters in the token though some would be repeated.

I think you're right that we have to handle short or long tokens.

I'd propose something like this:
Take the first 5 and last 5 characters if the token has at least 15 characters. If it has less than 5 characters remove all of the characters. We wouldn't want to just remove 1 character if we were actually trying to redact the token. If it has 5-15 characters we should remove the last 5 characters.

Also we need to indicate this is not the actual token value we received. We should include some filler indicator like "..." or "XXX" to make it clear it has been partially redacted.

Reply:
Thanks. 
1. I'm aware of the repeating character case. I thought it was ok :)
2. I only removed the last 1 character as a place holder to indicate that we need to sth about it. 
3. I was thinking about adding sth in between too.  This is good. now I don't have to worry about the response user gets back. Thank you!